style.css

```diff
@@ -0,0 +1,89 @@
+h1 {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
```

Comment:
- [ ] Right at the top of this h1 tag, I suggest you tag the body element and set the padding and margin to `0`, as you can see in the image below, the is a margin that makes your page have that small with space around, see the image below 🔽 

<img width="707" alt="Screen Shot 2022-02-10 at 23 25 38" src="https://user-images.githubusercontent.com/79058364/153500144-159aef05-8e74-4855-8a5e-ccf43b95996f.png">